src/Native/System.Native/pal_io.cpp

```diff
@@ -443,3 +464,93 @@ int32_t MksTemps(char* pathTemplate, int32_t suffixLength)
 {
     return mkstemps(pathTemplate, suffixLength);
 }
+
+static
+void ConvertPalPollFDToNative(const PollFD& pal, pollfd& native)
+{
+    native.fd = pal.FileDescriptor;
```

Comment:
For stat, we kept the names of the underlying struct members and just changed the casing e.g. FD, Events, REvents here. I think we should keep doing that to make it easier to correlate managed declarations with man pages.

Ditto for UTimBuf -> AcTime, ModTime.

Also, the convention for Convert functions so far has been (only one direction needed -> ConvertXxx, two directions -> ConvertXxxPalToPlatform, ConvertXxxPlatformToPal.